build/integration-in-docker.sh

```diff
@@ -58,12 +58,12 @@ function run_tests() {
     --privileged \
     --cap-add="sys_admin" \
     --entrypoint="" \
-    gcr.io/k8s-testimages/bootstrap \
+    gcr.io/k8s-staging-test-infra/bootstrap \
```

Comment:
FYI: You're going to have to switch this again soon when we move to artifact registry for staging images, and this image is also considered deprecated ...